src/main/pages/che-6/workspace-admin/recipes.adoc

```diff
@@ -156,50 +156,78 @@ The `env_file` Compose option is not supported. Environment variables can be man
 [id="kubernetes-yaml-limitations-and-restrictions"]
 == Kubernetes YAML limitations and restrictions
 
-When a workspace is starting, Che creates a https://kubernetes.io/docs/concepts/workloads/pods/pod/[Kubernetes pod]. The following are limitatons and restrictions:
+When a workspace is starting, Che creates various Kubernetes resources to support the IDE and development tools. Workspaces primarily consist of a https://kubernetes.io/docs/concepts/workloads/controllers/deployment/[Deployment] which runs a https://kubernetes.io/docs/concepts/workloads/pods/pod/[Kubernetes pod]. The following are limitatons and restrictions:
```

Comment:
@amisevsk maybe it would be a good idea to add info about replicas limitations that should only be `1` ?

Reply:
We could; I didn't add it because I think it's kind of common sense, and the validator will block stack creation, and currently KubernetesDeployments will directly set replicas to 1 if you happen to modify it.

I would rather keep the docs as short as possible and only cover the weird cases we require.